tests/SkiaSharp.NetCore.Tests/SkiaSharp.NetCore.Tests.csproj

```diff
@@ -12,18 +12,13 @@
     <PackageReference Include="xunit" Version="2.4.1" />
     <PackageReference Include="xunit.runner.visualstudio" Version="2.4.1" />
     <PackageReference Include="XunitXml.TestLogger" Version="2.1.26" />
-    <PackageReference Include="xunit.assemblyfixture" Version="2.0.3" />
```

Comment:
This never worked for .NET Core anyway. Use the xUnit-approved way from the xUnit samples.